roles/logging/tasks/main.yml

```diff
@@ -12,3 +12,8 @@
   notify: restart rsyslog
   when: logging.syslog_forwarding.enabled
 
+- include: monitoring.yml
+  tags: 
+    - monitoring
+    - sensu-checks
+  when: monitoring.enabled|default('True')|bool
```

Comment:
Why does this need to be renamed and moved to the new file? Is this a new style guide pattern we're following elsewhere, or is this simply to work around the fact this job is failing in CI and within CI monitoring is disabled?